README.md

```diff
@@ -71,6 +71,18 @@ wget https://raw.githubusercontent.com/gacopl/dvmkv2mp4/main/dvmkv2mp4
 chmod a+x dvmkv2mp4
 mv dvmkv2mp4 /usr/local/bin/
 ```
+
+To use dvmkv2mp4 in OS X:
+- download and install ionice-MacOS 
+- update dvmkv2mp4 by changing 
```

Comment:
Please update this comment and I'll merge many thanks!

Reply:
Comment updated